crates/systeminfo/src/hwinfo/mod.rs

```diff
@@ -45,6 +48,7 @@ impl HwInfo {
             cpu_boosting: self::cpufreq::get_cpu_boosting(),
             memory: Memory::new()?,
             network: self::net::get_interfaces(),
+            blocks: self::storage::get_blocks(),
```

Comment:
I don't think we should be calling this 'blocks' we know what this means, but in my opinion it's not great in the system info json to just have a list of things called 'blocks'. Not everybody will know what that means